test/live_select_tags_test.exs

```diff
@@ -274,8 +293,8 @@ defmodule LiveSelectTagsTest do
     select_nth_option(live, 2)
 
     assert_selected_multiple(live, [
-      %{label: "Rome", value: "Rome", tag_label: "R"},
-      %{label: "New York", value: "New York", tag_label: "NY"}
+      %{label: "Rome", value: "Rome", tag_label: "R", sticky: false},
+      %{label: "New York", value: "New York", tag_label: "NY", sticky: false}
     ])
```

Comment:
don't need to specify `sticky: false` here. We're only interested in checking that some options are selected and have a tag_label, we don't care about the sticky flag